package.json

```diff
@@ -66,6 +66,8 @@
   "private": true,
   "preferGlobal": false,
   "resolutions": {
+    "libp2p-delegated-content-routing": "0.3.1",
+    "libp2p-delegated-peer-routing": "0.3.1",
```

Comment:
This ensures fixes from https://github.com/libp2p/js-libp2p-delegated-peer-routing/pull/12 are in.